src/compiler/parse_table.cc

```diff
@@ -150,9 +151,7 @@ bool ParseState::has_shift_action() const {
 set<Symbol> ParseState::expected_inputs() const {
   set<Symbol> result;
   for (auto &entry : terminal_entries)
-    result.insert(Symbol(entry.first, true));
-  for (auto &entry : nonterminal_entries)
```

Comment:
This dates back from a time when the parse table was constructed differently. Prior to this commit, the only caller of `ParseState::expected_inputs()` would immediately filter out the nonterminals.